### PR TITLE
Changed requirements to pull edx-i18n-tools from PyPI and .gitignore …

### PR DESCRIPTION
**Pull request by Dillon-Dumesnil**
…to include en files

@clintonb @bderusha Please review

### CODE REVIEW

File: requirements/local.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ django-debug-toolbar==1.4
 
 # i18n
 transifex-client==0.11
-git+https://github.com/edx/i18n-tools.git@v0.1.4#egg=i18n_tools==0.1.4
+edx-i18n-tools>=0.3.3
```
Comment (nedbat) [inline requirements/local.txt]: I'm making the same comment in all the repos, sorry: don't we want to pin exact versions for IDA's?

**Comment by iivic** [comment]
@clintonb @mjfrey Please review.

**Comment by clintonb** [comment]
Include the English .po files.

**Comment by iivic** [comment]
Review once more.

**Comment by clintonb** [comment]
👍 (after .mo file added and commits squashed)

**Pull request merged**